README.md

```diff
@@ -1,76 +1,34 @@
 401 JS --  Lab 16 API AUTHORIZATION // LOGIN
 ===
 
-## Submission Instructions
-  * Work in a fork of this repository
-  * Work in a branch on your fork
-  * Write all of your code in a directory named `lab-` + `<your name>` **e.g.** `lab-duncan`
-  * Submit a pull request to this repository
-  * Submit a link to your pull request on canvas
-  * Submit a question, observation, and how long you spent on canvas  
-  
-## Learning Objectives  
-* Students will understand the difference between Hashing and Cypher algorithms and when they are used
-* Students will be able to implement basic auth for their APIs
+## Documentation
```

Comment:
I won't dock you for this the lacking README, but try to make them more robust in the future.